src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -548,16 +546,12 @@ private static bool IsNotPlayOffThemeAndBothTeamsNotNull(Game newGame, Tournamen
             string errorMessage = null;
             if (GameValidation.IsFreeDayGame(newGame))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the '?:' operator here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3240)